### ISSUE TITLE
Difference in whitespace when compiling IE9/10 media query hack

### ISSUE THREAD

**Issue by jvennix-rmn** [sass/libsass#1442]
An example is here: http://sassmeister.com/gist/3ad81feb2408d3b12298

Input:

```scss
@media screen and (min-width: 0\0) {
  body {
    color: blue;
  }
}
```

Output (libsass):

```css
@media screen and (min-width: 0 \0) {
  body {
    color: blue;
  }
}
```

Output (Ruby sass):

```css
@media screen and (min-width: 0\0) {
  body {
    color: blue;
  }
}
```

The Ruby version preserves the lack of whitespace in `0\0`. Adding the whitespace causes stripmq to fail when parsing the result CSS.

**Comment by xzyfer** [comment]
This is fixed in 3.3.

**Comment by jvennix-rmn** [comment]
Thanks!
